### PR TITLE
feat-issue-123: add rpc url env to replace default rpc url

### PR DESCRIPTION
**Pull request by fospring**
use env varibles to set rpc url
https://github.com/near/workspaces-js/issues/123

### CODE REVIEW

File: packages/js/src/utils.ts [hunk 1]
```diff
@@ -57,9 +57,16 @@ export function isTopLevelAccount(accountId: string): boolean {
 }
 
 function configFromDomain(network: 'testnet' | 'mainnet'): ClientConfig {
+  let rpcAddr = `https://archival-rpc.${network}.near.org`;
+  if (network === 'mainnet' && process.env.NEAR_CLI_MAINNET_RPC_SERVER_URL) {
+    rpcAddr = process.env.NEAR_CLI_MAINNET_RPC_SERVER_URL;
+  } else if (network === 'testnet' && process.env.NEAR_CLI_TESTNET_RPC_SERVER_URL) {
+    rpcAddr = process.env.NEAR_CLI_TESTNET_RPC_SERVER_URL;
+  }
+  console.log();
```
Comment (ailisp) [inline packages/js/src/utils.ts:66]:
```suggestion

```

File: packages/js/src/account/account-manager.ts [hunk 2]
```diff
@@ -217,7 +216,7 @@ export abstract class AccountManager implements NearAccountManager {
   }
 
   protected get connection(): nearAPI.Connection {
-    return new nearAPI.Connection(this.networkId, this.provider, this.signer);
+    return new nearAPI.Connection(this.networkId, this.provider, this.signer, `jsvm.${this.networkId}`);
```
Comment (ailisp) [inline packages/js/src/account/account-manager.ts:219]: What is this for?

**Comment by ailisp** [comment]
CI fails need to be fixed. I recently tried main branch can pass: https://github.com/near/workspaces-js/pull/218

File: packages/js/src/utils.ts (hunk 1 above)
Comment (fospring) [reply packages/js/src/utils.ts:66]: ok

File: packages/js/src/account/account-manager.ts (hunk 2 above)
Comment (fospring) [reply packages/js/src/account/account-manager.ts]: I used the incorrect near api js version cached on my mechine, so I add a new param to run test on my local mechine, I will changed it

**Review by ailisp — APPROVED** [review]
Nice enhancement!

**Pull request merged**